TumbleBitSetup/Utils.cs

```diff
@@ -55,7 +60,52 @@ internal static int GetOctetLen(int x)
         {
             return (int)Math.Ceiling((1.0 / 8.0) * Math.Log(x+1, 2));
         }
-        
+
+        internal static AsymmetricCipherKeyPair GeneratePrivate(BigInteger exp, int keySize)
+        {
+            SecureRandom random = new SecureRandom();
+            var gen = new RsaKeyPairGenerator();
+            gen.Init(new RsaKeyGenerationParameters(exp, random, keySize, 2)); // See A.15.2 IEEE P1363 v2 D1 for certainty parameter
```

Comment:
Oops, I think that's my fault, I copied it from TumbleBit to RsaKey.cs (Nicolas just moved it from RsaKey.cs to here) but I forgot to update it after the fix. I'll push a commit with the fix after I merge.